packages/breadcrumbs/src/elements/Breadcrumb.tsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { Children, cloneElement, HTMLAttributes } from 'react';
+import { useBreadcrumb } from '@zendeskgarden/container-breadcrumb';
+
+import { StyledBreadcrumb, StyledBreadcrumbItem } from '../styled';
+
+export const Breadcrumb: React.FC<HTMLAttributes<HTMLOListElement>> = React.forwardRef(
```

Comment:
This is downcasting the `forwardRef` value and removes the ability for the `ref` to be used. `forwardRef` provides some generics for us to use. Try:

```tsx
export const Breadcrumb = React.fowardRef<HTMLElement, HTMLAttributes<HTMLElement>((props, ref) => {});
```

This will type the props and ref value correctly.

Reply:
Also, now that the props are spread onto a `<nav>` element I believe `HTMLElement` is the correct base-type, not `HTMLOListElement`

Reply:
Hmmm, thanks I'll give that a shot!

Reply:
Thanks for the suggestion! This makes sense.